packages/front-end/src/app/components/EGRunPipelineFormRunDetails.vue

```diff
@@ -30,7 +31,14 @@
    * viralrecon-illumina_community-showcase_20240712_5686910e783b4b2
    */
   const MAX_TOTAL_LENGTH = 80;
-  const maxRunNameLength = ref(MAX_TOTAL_LENGTH - props.pipelineName.length - 8 - 15 - 2);
+  const maxRunNameLength = computed<number>(() => {
```

Comment:
Just be mindful a change when in yesterday that will override this when you resolve the merge conflicts - it should set the value at a constant 50 characters.

Reply:
just merged and have preserved the 50 char constant 👍